titus-common/src/main/java/com/netflix/titus/common/util/limiter/Limiters.java

```diff
@@ -75,4 +80,54 @@ public static TokenBucket createInstrumentedFixedIntervalTokenBucket(String name
                 titusRuntime
         );
     }
+
+    private static class Unlimited implements TokenBucket {
+        private final String name;
+
+        public Unlimited(String name) {
+            this.name = name;
+        }
+
+        @Override
+        public String getName() {
+            return name;
+        }
+
+        @Override
+        public long getCapacity() {
+            return Long.MAX_VALUE;
+        }
+
+        @Override
+        public long getNumberOfTokens() {
+            return Long.MAX_VALUE;
+        }
+
+        @Override
+        public boolean tryTake() {
+            return true;
+        }
+
+        @Override
+        public boolean tryTake(long numberOfTokens) {
+            return true;
+        }
+
+        @Override
+        public void take() {
+        }
+
+        @Override
+        public void take(long numberOfTokens) {
+        }
+
+        @Override
+        public void refill(long numberOfToken) {
+        }
+
+        @Override
+        public RefillStrategy getRefillStrategy() {
+            return null;
```

Comment:
We should return an actual strategy here.

Reply:
Yeah, thanks for keeping me honest. I was being lazy here. I will add it